diff --git a/nix/nimble.nix b/nix/nimble.nix
index 5343aaa81..770f8fa5a 100644
--- a/nix/nimble.nix
+++ b/nix/nimble.nix
@@ -8,5 +8,5 @@ in pkgs.fetchFromGitHub {
   repo = "nimble";
   rev = tools.findKeyValue "^ +NimbleStableCommit = \"([a-f0-9]+)\".+" sourceFile;
   # WARNING: Requires manual updates when Nim compiler version changes.
-  hash = "sha256-sa0irAZjQRZLduEMBPf7sHlY1FigBJTR/vIH4ihii/w=";
+  hash = "sha256-MVHf19UbOWk8Zba2scj06PxdYYOJA6OXrVyDQ9Ku6Us=";
 }
